agents/mongodb/internal/profiler/profiler_test.go

```diff
@@ -105,15 +105,15 @@ func TestProfiler(t *testing.T) {
 		fieldsCount := dbNumber + 1
 		doc := bson.M{}
 		for j := 0; j < fieldsCount; j++ {
-			doc[fmt.Sprintf("name_%02d\xff", j)] = fmt.Sprintf("value_%02d\xff", j) // to generate different fingerprints
```

Comment:
this was done for the reason. Why did you remove it?

Reply:
Didn't know it was an UTF8 test. I thought it was a copy of a random data generator.
Restored the test.